proto/v1/structure/question.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package org.oppia.proto.v1.structure;
+
+import "proto/v1/structure/image.proto";
+import "proto/v1/structure/state.proto";
+import "proto/v1/structure/versions.proto";
+
+option java_package = "org.oppia.proto.v1.structure";
+option java_multiple_files = true;
+
+// Structure of the single question. 
+message Question {
+  QuestionStructureVersion structure_version = 1;
+
+
+  // Id of the question.
+  string id = 2;
+
+  // Question state is the name of the state to which this question belongs.
+  State question_state = 3;
+
+  // All the skill ids which belongs to this question.
+  repeated string linked_skill_ids = 4;
+
+  int32 content_version = 5;
+  ReferencedImageList referenced_image_list = 6;
```

Comment:
The list of all images that are included within SubtitledHTML subfields of this question.

Reply:
done